tools/cli/workflowCommands.go

```diff
@@ -970,13 +970,22 @@ func convertFailure(failure *failurepb.Failure) *clispb.Failure {
 		return nil
 	}
 
-	return &clispb.Failure{
+	var fType string
+	if failure.GetTimeoutFailureInfo() != nil {
+		fType = failure.GetTimeoutFailureInfo().GetTimeoutType().String()
+	} else {
+		fType = reflect.TypeOf(failure.GetFailureInfo()).Elem().Name()
+	}
```

Comment:
```suggestion
	fType := reflect.TypeOf(failure.GetFailureInfo()).Elem().Name()
	if failure.GetTimeoutFailureInfo() != nil {
		fType = fmt.Sprinf("%s: %s", fType, failure.GetTimeoutFailureInfo().GetTimeoutType().String())
	}
```
Let's keep failure type (Timeout) there.